build.psm1

```diff
@@ -973,10 +975,17 @@ function Install-Dotnet {
 }
 
 function Get-RedHatPackageManager {
+    param(
+        [Parameter(Mandatory=$true)]
+        [switch]
+        $install
```

Comment:
Right now we are only getting an install command.  
I can remove it if you think that is a better alternative.